ts/boot/configureStoreAndPersistor.ts

```diff
@@ -177,6 +176,18 @@ const migrations: MigrationManifest = {
         servicesByScope: pot.none
       }
     };
+  },
+
+  // Version 10
+  // we convert the acknowledgeOnEmailValidated state type from Option<boolean> to boolean
+  "10": (state: PersistedState) => {
```

Comment:
I checked (but check you too) **EmailValidationState** is not persisted
So the shape of the state could be changed without a migration

Reply:
updated within the commited code